lib/practice_exercises.rb

```diff
@@ -1,13 +1,22 @@
 
-# Time Complexity: ?
-# Space Complexity: ?
+# Time Complexity: O(n^2)
```

Comment:
That is the correct time complexity given that you used `delete_at`, Is there a faster way?